clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -1294,7 +1294,7 @@ boolean generationUnchanged() {
         }
     }
 
-    protected final Meter createMeter(Metrics metrics, String groupName, String baseName, String descriptiveName) {
+    static Meter createMeter(Metrics metrics, String groupName, String baseName, String descriptiveName) {
```

Comment:
It is a bit weird to just expose it like this. `createMeter` does not mean much outside of the usages within this class and sub-class. We could perhaps find a better place for this one and also find a better name. What do you think?

Reply:
Agreed.

I've updated how this is done in the refactoring PR #14565.